docs/language_specification/statements/gates.md

```diff
@@ -0,0 +1,163 @@
+Gates define single-qubit or multi-qubit unitary operations
+that change the state of a qubit register in a deterministic fashion.
```

Comment:
I would avoid the use of deterministic here. I would suggest:

Gates define single-qubit or multi-qubit unitary operations
that modify the state of a qubit register.

Reply:
Sorry for the confusion, but this change doesn't belong to this PR. Can you comment on PR https://github.com/QuTech-Delft/cQASM-spec/pull/47 as well? Thanks!